xen/arch/arm/cpufreq/arm_scpi.c

```diff
@@ -1036,9 +1038,8 @@ static int scpi_probe(struct platform_device *pdev)
 		cl->dev = dev;
 		cl->rx_callback = scpi_handle_remote_msg;
 		cl->tx_prepare = scpi_tx_prepare;
-		cl->tx_block = true;
-		cl->tx_tout = 20;
-		cl->knows_txdone = false; /* controller can't ack */
+		cl->tx_block = false;
+		cl->knows_txdone = true;
```

Comment:
Can't this stuff be configurable through the device tree?

Reply:
I really don't know. Probably not with generic mailbox bindings. Probably yes with custom bindings.

Reply:
There is no such binding for saying client to block or don't block.